src/core/Akka/Dispatch/Dispatchers.cs

```diff
@@ -262,7 +262,7 @@ public override void Execute(IRunnable run)
         {
             if (Volatile.Read(ref _shuttingDown) == 1)
                 throw new RejectedExecutionException("ForkJoinExecutor is shutting down");
-            _dedicatedThreadPool.QueueUserWorkItem(run.Run);
+            _dedicatedThreadPool.QueueUserWorkItem(r=>r.Run(), run);
```

Comment:
Frankly, we would be better off here if we just took `IRunnable` in here and called .Run() directly, which would be a good next step. That would eliminate a bit more indirection as well as further improve allocations.